src/main/java/org/sonarqube/gradle/SonarQubeTask.java

```diff
@@ -38,32 +43,61 @@
  */
 public class SonarQubeTask extends DefaultTask {
 
-  private Map<String, Object> sonarProperties;
+    private static final Logger LOGGER = Logging.getLogger(SonarQubeTask.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)